packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -2720,6 +2723,52 @@ async function handler(
   const metadataFromEvent = videoCallUrl ? { videoCallUrl } : undefined;
 
   try {
+    const hasExistingWorkflow: boolean = eventType.workflows.some((workflow) => {
```

Comment:
Can we create a helper function for this (in workflows/lib/reminders) which we can also be reused in handleConfirmation?